pkg/controller/controller_instance_test.go

```diff
@@ -115,42 +115,34 @@ func TestReconcileInstanceWithAuthError(t *testing.T) {
 
 	testController.reconcileInstance(instance)
 
+	// verify that one catalog client action occurred
 	actions := fakeCatalogClient.Actions()
-	assertNumberOfActions(t, actions, 1)
```

Comment:
Oh, weird.  Hrm.... i tell you what, I'm not quite convinced yet on the catalog actions -- the rest I'm totally for.  Doing the kube actions and events will add quality - how about we just do those for now?